CHANGELOG.rst

```diff
@@ -9,12 +9,14 @@ Since last release
 
 **Changed:**
 
-* Rely on `python3` in environment instead of `python` (#602)
+* Rely on ``python3`` in environment instead of ``python`` (#602)

```

Comment:
Curious to know what this is about?

Reply:
Mostly aesthetic preference.  I have a habit of using single backticks (Markdown syntax) for inline code formatting, but RST syntax is double backticks.